presto-product-tests/conf/presto/etc/file_auth_copy

```diff
@@ -0,0 +1,2 @@
+userPBKDF2:1000:5b4240333032306164:f38d165fce8ce42f59d366139ef5d9e1ca1247f0e06e503ee1a611dd9ec40876bb5edb8409f5abe5504aab6628e70cfb3d3a18e99d70357d295002c3d0a308a0
```

Comment:
why do we need to copy this?

Reply:
ie remove the `_copy` from the repo

Reply:
I created a copy and use it in the tests to avoid duplicate entries in case of product test failures.
 For example, if the product test fails just after adding in the original file (the delete part is not executed), during the next run the add part will again add the same entries  leading to duplicate entries.